### DISCUSSION TITLE
How to manually configure the model path？

### DISCUSSION THREAD

**Discussion by bouyeijiang** [PaddlePaddle/PaddleFormers#2040] [Q&A] [answered]
hub.Module(name="chinese_ocr_db_crnn_mobile")

I need to change the model save path

**Comment by rainyfly** [comment, marked as answer]
All module is saved in ~/.paddlehub/modules, you can find the module under this path. For paddlehub, if you move the module directory to other path, you can use hub.Module(directory= 'chinese_ocr_db_crnn_mobile') to load it.
